### ISSUE TITLE
jOOQ-meta column addition fails with `references` foreign key syntax

### ISSUE THREAD

**Issue by thyming** [jOOQ/jOOQ#9615]
Hello, I was giving jOOQ-meta a try and came across what appears to be a bug. Here's a simple reproduction using jooq 3.12.3:
```sql
create table a
(
    id int primary key
);

create table b
(
    id int primary key
);

alter table b
    add column a_id int references a;
```
This is (at the very least) valid syntax in PostgreSQL. The same error occurs if the primary key of a is reference explicitly, like
```sql
alter table b
    add column a_id int references a(id);
```
The equivalent, as a workaround, works:
```sql
alter table b
    add column a_id int;

alter table b
    add foreign key (a_id) references a;
```

The full stack trace:
```
Dec 03, 2019 10:42:59 AM org.jooq.tools.JooqLogger error
SEVERE: Cannot read {...}/generateMainJooqSchemaSource/config.xml. Error : Unexpected token or missing query delimiter: [12:25] ...ary key
);

alter table b
    add column a_id int [*]references a;

org.jooq.impl.ParserException: Unexpected token or missing query delimiter: [12:25] ...ary key
);

alter table b
    add column a_id int [*]references a;

        at org.jooq.impl.ParserContext.exception(ParserImpl.java:10675)
        at org.jooq.impl.ParserContext.done(ParserImpl.java:10795)
        at org.jooq.impl.ParserImpl.parse(ParserImpl.java:550)
        at org.jooq.impl.ParserImpl.parse(ParserImpl.java:529)
        at org.jooq.meta.extensions.ddl.DDLDatabase.load(DDLDatabase.java:188)
        at org.jooq.meta.extensions.ddl.DDLDatabase.access$100(DDLDatabase.java:91)
        at org.jooq.meta.extensions.ddl.DDLDatabase$2.load(DDLDatabase.java:169)
        at org.jooq.meta.tools.FilePattern.load(FilePattern.java:165)
        at org.jooq.meta.tools.FilePattern.load(FilePattern.java:178)
        at org.jooq.meta.tools.FilePattern.load(FilePattern.java:138)
        at org.jooq.meta.extensions.ddl.DDLDatabase.create0(DDLDatabase.java:166)
        at org.jooq.meta.AbstractDatabase.create(AbstractDatabase.java:260)
        at org.jooq.meta.AbstractDatabase.create(AbstractDatabase.java:249)
        at org.jooq.meta.AbstractDatabase.setConnection(AbstractDatabase.java:239)
        at org.jooq.codegen.GenerationTool.run0(GenerationTool.java:501)
        at org.jooq.codegen.GenerationTool.run(GenerationTool.java:221)
        at org.jooq.codegen.GenerationTool.generate(GenerationTool.java:216)
        at org.jooq.codegen.GenerationTool.main(GenerationTool.java:188)
```

Let me know if I can provide any more information or help!

**Comment by lukaseder** [comment]
Thanks a lot for your suggestion. Indeed, we unfortunately do not support this syntax yet.

I'm closing this as a duplicate of: https://github.com/jOOQ/jOOQ/issues/5317

**Comment by lukaseder** [comment]
https://github.com/jOOQ/jOOQ/issues/5317 is implemented for jOOQ 3.13. The change is quite significant, so we're not going to backport it to 3.12.x.

Thanks again for your report.

**Comment by thyming** [comment]
Thanks!!
